CHANGELOG.md

```diff
@@ -2,18 +2,19 @@
 
 SECURITY:
 
- * build: Updated to Go 1.14.6. Go 1.14.4 contained 2 CVEs which are low severity for Nomad. [[GH-8467](https://github.com/hashicorp/nomad/issues/8467)]
+ * build: Updated to Go 1.14.6. Go 1.14.5 contained 2 CVEs which are low severity for Nomad [[GH-8467](https://github.com/hashicorp/nomad/issues/8467)]
```

Comment:
It's golang 1.14.5 that had the CVE: https://golang.org/doc/devel/release.html#go1.14.minor

>  go1.14.5 (released 2020/07/14) includes security fixes to the crypto/x509 and net/http packages. See the [Go 1.14.5 milestone](https://github.com/golang/go/issues?q=milestone%3AGo1.14.5+label%3ACherryPickApproved) on our issue tracker for details.